config/testdata/conf.good.yml

```diff
@@ -121,6 +121,9 @@ scrape_configs:
   marathon_sd_configs:
   - servers:
     - 'http://marathon.example.com:8080'
+    tls_config:
```

Comment:
Yes, thank you. I will replace the url with something like 'https://marathon.example.com:443'